bluebottle/payouts_dorado/adapters.py

```diff
@@ -23,11 +32,15 @@ def trigger_payout(self):
 
         try:
             response = requests.post(self.settings['url'], data)
-            if response.content != '{"status": "success"}':
-                raise SystemError("Could not trigger payout")
-            #
+            response.raise_for_status()
+
             self.project.payout_status = 'created'
             self.project.save()
+        except requests.HTTPError:
+            try:
+                raise PayoutValidationError(json.loads(response.content))
+            except ValueError:
+                raise PayoutCreationError(response.content)
```

Comment:
If there is an payout creation error, we could reset the payout_status to its previous state.